masterlist.yaml

```diff
@@ -6769,6 +6769,47 @@ plugins:
       # version: 1.06a
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
+    group: *lowPriorityGroup
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/15649/' ]
+    msg:
+      - <<: *patchIncluded
+        subs: [ 'Mortal Enemies' ]
+        condition: 'active("Mortal Enemies.esp") and not active("Immersive Movement - Mortal Enemies Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Smilodon' ]
+        condition: 'active("Smilodon - Combat of Skyrim.esp") and not active("Immersive Movement - Smilodon Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Wildcat' ]
+        condition: 'active("Wildcat - Combat of Skyrim.esp") and not active("Immersive Movement - Wildcat Patch.esp")'
+    clean:
+      # version: 1.3.2
+      - crc: 0xB91C4616
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
```

Comment:
`Req:` would be better for `Immersive Movement.esp`.

Reply:
Done.